exercises/concept/spellbound-steel/.docs/instructions.md

```diff
@@ -39,14 +39,14 @@ SpellboundSteel.capitalize_cards(cards)
 In the game is it important to calculate the power level of all cards.
 The power for all cards can been seen in the following table:
 
-| Card Type | Power Level |
-| --------- | ----------- |
-| Warrior   | 10          |
-| Mage      | 20          |
-| Rogue     | 30          |
-| Fireball  | 15          |
-| Ice Storm | 25          |
-| Lightning Bolt | 35    |
+| Card Type      | Power Level |
+|----------------|-------------|
+| Warrior        | 10          |
+| Mage           | 20          |
+| Rogue          | 30          |
+| Fireball       | 15          |
+| Ice Storm      | 25          |
+| Lightning Bolt | 35          |
```

Comment:
I use a markdown formatter that makes makes tables easier to read in their raw format. I can revert this if we'd prefer to not have this change.